scripts/globals/job_utils/dancer.lua

```diff
@@ -29,6 +30,9 @@ local function getStepFinishingMovesBase(player)
         numAwardedMoves = 2
     end
 
+    numAwardedMoves = numAwardedMoves + player:getMod(xi.mod.STEP_FINISH)
```

Comment:
Seems reasonable; however, since this mod is only used by Terpsichore, can anyone confirm the behavior with it regarding presto?  Is it still clamped, or will that give 6 moves?

Reply:
![imagen](https://user-images.githubusercontent.com/60053999/217973505-d0e13f9d-9099-46d5-8a9b-064dd28a387b.png)
Would need to also test if the Weapon is indeed needed in the main slot or not

Reply:
> Seems reasonable; however, since this mod is only used by Terpsichore, can anyone confirm the behavior with it regarding presto? Is it still clamped, or will that give 6 moves?

I believe the max moves you can have is 5 for this era, correct?